src/System.Net.Security/tests/FunctionalTests/SslStreamAlpnTests.cs

```diff
@@ -115,14 +116,14 @@ public void SslStream_StreamToStream_Alpn_Success(List<SslApplicationProtocol> c
         }
 
         [Fact]
-        [PlatformSpecific(~TestPlatforms.OSX)]
+        [PlatformSpecific(TestPlatforms.Linux)]
         public void SslStream_StreamToStream_Alpn_NonMatchingProtocols_Fail()
```

Comment:
@Drawaes Why are you excluding this test from Windows? This should run on all OSes except OSX.

Reply:
because now it doesn't work, I am trying to get the maximum to pass, 

this test as far as I can tell never did anything on any platform before because the tasks were not awaited.